src/main/java/WayofTime/bloodmagic/api/event/ItemBindEvent.java

```diff
@@ -5,6 +5,15 @@
 import net.minecraftforge.fml.common.eventhandler.Cancelable;
 import net.minecraftforge.fml.common.eventhandler.Event;
 
+/**
```

Comment:
Standardize the placement of these javadocs. This one is on the class and SacrificialKnifeUsedEvent is on the constructor.